docs/spot-instances.md

```diff
@@ -83,6 +83,29 @@ Launch Template support is a recent addition to both AWS and this module. It mig
   ]
 ```
 
+Example launch template to use spot options:
+
+```hcl
+  worker_groups_launch_template = [
+    {
+      name                             = "spot-1"
+      instance_type                    = "t3.large"
+      market_type                      = "spot"
+      spot_options                     = {
+        block_duration_minutes         = "60"
+        instance_interruption_behavior = "terminate"
+        max_price                      = null
```

Comment:
code should be able to take a default provider values in cae of missing value for field. As a lot of them are optional we cannot require to put `null` from users. 

Here you can take a look of some constructions which we using to make such optional inputs:
https://github.com/terraform-aws-modules/terraform-aws-eks/blob/56e93d77de58f311f1d1d7051f40bf77e7b03524/modules/node_groups/launch_template.tf#L88-L92

https://github.com/terraform-aws-modules/terraform-aws-eks/blob/56e93d77de58f311f1d1d7051f40bf77e7b03524/modules/fargate/main.tf#L66-L69

Reply:
Fixed.